src/services/container.ts

```diff
@@ -57,11 +58,16 @@ async function clearDomainCookies(
 ): Promise<void> {
   const containers = await browser.contextualIdentities.query({})
 
-  const otherContainers = containers.filter(
-    (container) => container.cookieStoreId !== cookieStoreId
-  )
+  const cookieStoreToBeRemoved = containers
+    .filter((container) => container.cookieStoreId !== cookieStoreId)
+    .map((container) => ({
+      cookieStoreId: container.cookieStoreId
+    }))
+    .concat({
+      cookieStoreId: DEFAULT_FIREFOX_COOKIE_STORE_ID
```

Comment:
how about we just extract the `cookieStoreId`? the reason is the type may not be correct, other field will all become optional because of this line

Reply:
While using map above, the type hsas already convert to with cookieStoreId only. do u mean we should keep the other fields?

Reply:
@nickccm1122 I see, why don't we just use `Array<string>`, seems the object is unnecessary

Reply:
@foray1010 updated